docs/resources/anycast_host.md

```diff
@@ -0,0 +1,176 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "bloxone_anycast_host Resource - terraform-provider-bloxone"
+subcategory: "Anycast"
+description: |-
+  Retrieve an Anycast host Configurations.
+---
+
+# bloxone_anycast_host (Resource)
+
+Retrieve an Anycast host Configurations.
+
+## Example Usage
+
+```terraform
+data "bloxone_infra_hosts" "anycast_hosts" {
+  filters = {
+    display_name = "anycast_real"
+  }
+}
+
+# Create an anycast config profile with onprem hosts
+resource "bloxone_anycast_config" "test_onprem_hosts" {
+  anycast_ip_address = "10.10.10.1"
+  name               = "Anycast_config_example"
+  service            = "DNS"
+
+}
+
+# Adding an anycast host with BGP routing protocol
+resource "bloxone_anycast_host" "test_anycast_host" {
+  id = one(data.bloxone_infra_hosts.anycast_hosts.results).legacy_id
+
+  # Adding the anycast config profile and enabling BGP routing protocol
+  anycast_config_refs = [
+    {
+      anycast_config_name = bloxone_anycast_config.test_onprem_hosts.name
+      routing_protocols   = ["BGP", "OSPF"]
+    }
+  ]
+
+  # Adding the BGP configuration
+  config_bgp = {
+    asn           = "6500"
+    asn_text      = "6500"
+    holddown_secs = 180
+    neighbors = [
+      {
+        asn        = "6501"
+        ip_address = "10.20.0.3"
+      }
+    ]
+  }
+
+  # Adding the BGP configuration
+  config_ospf = {
+    area_type           = "STANDARD"
+    area                = "10.0.0.1"
+    authentication_type = "Clear"
+    interface           = "eth0"
+    authentication_key  = "YXV0aGV"
+    hello_interval      = 10
+    dead_interval       = 40
+    retransmit_interval = 5
+    transmit_delay      = 1
+  }
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Optional
+
+- `anycast_config_refs` (Attributes List) (see [below for nested schema](#nestedatt--anycast_config_refs))
+- `config_bgp` (Attributes) (see [below for nested schema](#nestedatt--config_bgp))
+- `config_ospf` (Attributes) (see [below for nested schema](#nestedatt--config_ospf))
+- `config_ospfv3` (Attributes) (see [below for nested schema](#nestedatt--config_ospfv3))
+
+### Read-Only
+
+- `created_at` (String)
+- `id` (Number) The ID of this resource.
+- `ip_address` (String) IPv4 address of the on-prem host
+- `ipv6_address` (String) IPv6 address of the on-prem host
+- `name` (String)
+- `updated_at` (String)
+
+<a id="nestedatt--anycast_config_refs"></a>
+### Nested Schema for `anycast_config_refs`
+
+Required:
+
+- `anycast_config_name` (String)
+
+Optional:
+
+- `routing_protocols` (List of String) Routing protocols enabled for this anycast configuration, on a particular host. Valid protocol names are "BGP", "OSPF"/"OSPFv2", "OSPFv3".
+
+
+<a id="nestedatt--config_bgp"></a>
+### Nested Schema for `config_bgp`
+
+Required:
+
+- `asn` (Number)
+- `holddown_secs` (Number)
```

Comment:
missing description, there are other fields as well that are missing description. Can you please add description for all?

Reply:
done